src/features/unmarkedWsNoticeList/printAllStickersButton/PrintAllStickersButton.tsx

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
-import { useQuery } from '@apollo/react-hooks';
+import { useMutation, useQuery } from '@apollo/react-hooks';
 import { useTranslation } from 'react-i18next';
 
 import Button from '../../../common/button/Button';
 import { UNMARKED_WINTER_STORAGE_NOTICES_STICKERS_QUERY } from './queries';
 import { generateAndSaveStickerPDF, getCustomersWithStickers } from '../utils';
 import { UNMARKED_WINTER_STORAGE_NOTICES_STICKERS } from './__generated__/UNMARKED_WINTER_STORAGE_NOTICES_STICKERS';
 import styles from './printAllStickersButton.module.scss';
+import {
+  SET_STICKERS_POSTED,
+  SET_STICKERS_POSTEDVariables as SET_STICKERS_POSTED_VARS,
+} from '../__generated__/SET_STICKERS_POSTED';
+import { SET_STICKERS_POSTED_MUTATION } from '../mutations';
 
 const PrintAllStickersButton = () => {
   const { t } = useTranslation();
   const { loading, data } = useQuery<UNMARKED_WINTER_STORAGE_NOTICES_STICKERS>(
     UNMARKED_WINTER_STORAGE_NOTICES_STICKERS_QUERY
   );
+  const [setStickersPosted] = useMutation<SET_STICKERS_POSTED, SET_STICKERS_POSTED_VARS>(SET_STICKERS_POSTED_MUTATION);
+
+  const onClick = () => {
+    const customers = getCustomersWithStickers(data);
+    generateAndSaveStickerPDF(customers);
+    setStickersPosted({
+      variables: {
+        input: {
+          leaseIds: customers.map((customer) => customer.leaseId as string),
```

Comment:
What if the leaseId is `undefined`?

Reply:
Those have been filtered out out `getCustomersWithStickers`. If it has a sticker number it has a lease, and if it has a lease the leaseId cannot be null per the schema.

Reply:
Fair enough! 👍